kernel/kernel-api/src/test/scala/io/delta/kernel/internal/tablefeatures/TableFeaturesSuite.scala

```diff
@@ -230,115 +245,316 @@ class TableFeaturesSuite extends AnyFunSuite {
   })
 
   /////////////////////////////////////////////////////////////////////////////////////////////////
-  // Legacy tests (will be modified or deleted in subsequent PRs)                                //
```

Comment:
test case for some unknown table feature? also protocol versions > 3,7?

Reply:
correct me if I'm just not seeing it

Reply:
added